opencti-platform/opencti-graphql/src/manager/taskManager.js

```diff
@@ -242,32 +243,24 @@ const executeRemove = async (context, user, actionContext, element) => {
 export const executeReplace = async (context, user, actionContext, element) => {
   const { field, type: contextType, values } = actionContext;
   if (contextType === ACTION_TYPE_RELATION) {
-    const toCreate = [];
-    // 01 - Delete all relations of the element
+    // 01 - Get all relations of the element
     const rels = await listAllRelations(context, user, field, { fromId: element.id });
+    // Patch
     if (rels.length > 0) {
-      // Items to Delete
       for (let index = 0; index < rels.length; index += 1) {
         if (!values.includes(rels[index].toId)) {
-          const rel = rels[index];
-          await deleteElementById(context, user, rel.id, rel.entity_type);
-        }
-      }
-      // 02 - Create new ones
-      for (let indexCreate = 0; indexCreate < values.length; indexCreate += 1) {
-        const target = values[indexCreate];
-        if (!rels.some(({ toId }) => toId === target)) {
-          toCreate.push(target);
+          const inputName = schemaRelationsRefDefinition.convertDatabaseNameToInputName(element.entity_type, field);
```

Comment:
`values` is dataBaseName style in `actionContext`, but `generatePatch` needs inputName style.
Do we need to change` values` in `actionContext`?